mediation_fees.rst

```diff
@@ -53,7 +53,7 @@ Fee calculation
 
 There are two fundamental formula to relate :math:`a`, :math:`b` and :math:`c`.
 
-1. :math:`a = c + aq + f + i(a)`
+1. :math:`a = c + aq + f + i(-a)`
```

Comment:
A small comment here to explain why `i(-a)` is used would be nice, but we can also do this later